adhocteam/Head-Start-TTADP
# Display my alerts

**Description of change**
Displays alerts for managers, authors and collaborators. If there are no alerts, an empty area will be displayed with the text "You're all caught up" (per design).

**How to test**
- pull changes
- enter a couple of reports
- note that the reports show up in both sections, the "My activity report alerts" and "Activity reports"
- modify .env file to simulate a different user (one that is not a manager or a collaborator)
- restart the backend server
- note that the reports only show up in the lower table and there is a "You're all caught up" section.
- modify .env file to simulate a user that is either a manager or a collaborator on existing reports
- restart the backend server
- verify that the corresponding reports are displayed in the 

**Issue(s)**
* https://github.com/HHS/Head-Start-TTADP/issues/181
* https://github.com/HHS/Head-Start-TTADP/issues/191
* https://github.com/HHS/Head-Start-TTADP/issues/192

**Checklist**

    - API methods
    - Boundary and Data Flow Diagrams
    - [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions with the [Nygard template](https://github.com/joelparkerhenderson/architecture_decision_record/blob/master/adr_template_by_michael_nygard.md)
    - OSCAL templates completed when security controls are implemented or modified
